topics/icrc_33_call_canister.md

```diff
@@ -0,0 +1,191 @@
+# ICRC-33: Call Canister
+
+| Status |
+|:------:|
+| Draft  |
+
+<!-- TOC -->
+* [ICRC-33: Call Canister Standard](#icrc-33-call-canister)
+    * [Summary](#summary)
+    * [Scope](#scope)
+    * [Method](#method)
+    * [Request](#request)
+        * [Example RPC Request ](#example-rpc-request)
+    * [Response](#response)
+        * [Example RPC Response ](#example-rpc-response)
+    * [Message Processing](#message-processing)
+    * [Errors](#errors)
+
+## Summary
+
+This standard defines a method for requesting 3rd party canister calls to be executed by the signer utilising a user selected identity.
+
+This message can be used by the relying party to request canister calls to be executed by the signer.
```

Comment:
The second sentence feels redundant? However, I would add some words about user approval / transaction approval to explain why this is secure.

Reply:
Oh, yea. Will merge the two